<launch>

<node pkg="robot_pose_ekf" type="robot_pose_ekf" name="robot_pose_ekf" output="screen">
  <param name="output_frame" value="odom"/>
  <param name="freq" value="30.0"/>
  <param name="debug" value="true"/>
  <param name="sensor_timeout" value="1.0"/>  
  <param name="odom_used" value="true"/>
  <param name="imu_used" value="true"/>
  <param name="vo_used" value="false"/>
  <param name="gps_used" value="true"/>

</node>


</launch>

